java/src/jmri/jmrit/decoderdefn/IdentifyDecoder.java

```diff
@@ -23,7 +23,9 @@
  * models, in which case no "productID" can be determined. (This code uses
  * {@link #setOptionalCv(boolean flag) setOptionalCv()} and
  * {@link #isOptionalCv() isOptionalCv()} as documented below.)</li>
- * <li>TCS: (mfgID == 153) CV249 is ID</li>
+ * <li>TCS: (mfgID == 153) CV249 is physical hardware id, V5 and above use
+ * CV253, CV254, CV255, and CV256 to identify specific sound sets and 
+ * features. New productID process triggers if (CV249 > 175).</li>
```

Comment:
I think `(CV249 > 175)` needs to be written as `(CV249 &gt; 175)` since Javadoc is converted to HTML.

Static analys failed with:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.0:javadoc (default-cli) on project jmri: An error has occurred in Javadoc report generation: 
Error:  Exit code: 1 - /home/runner/work/JMRI/JMRI/java/src/jmri/jmrit/decoderdefn/IdentifyDecoder.java:28: error: bad use of '>'
Error:   * features. New productID process triggers if (CV249 > 175).</li>
```

Reply:
@TCSDCC-Jordan
The PR has an error that needs to be fixed. If you want, I can do the change for you.

Reply:
@danielb987 
Thanks for the heads up. I made the conversion over to &gt; 
Do I need to re-submit my pull-request or can it be merged in from this still?

Reply:
@TCSDCC-Jordan
> Do I need to re-submit my pull-request or can it be merged in from this still?

No need to re-submit. This can be merged.